gallery/rediscache-insights/Azure Monitor.json

```diff
@@ -3,11 +3,11 @@
 	"version": "TemplateGallery/1.0",
 	"categories": [
 		{
-			"id": "Insights",
-			"name": "Insights",
+			"id": "RedisCache",
+			"name": "Azure Cache for Redis Insights",
```

Comment:
For the applicationinsights-devtools team, I'm happy to undo this `id` + `name` change if it's preferable not to introduce a new category. The motivation for the change was to match other similar categories like `{ "id": "CosmosDb", "name": "Azure Cosmos DB" }`, but no worries if it's better to keep the original category.